drydock_provisioner/drydock.py

```diff
@@ -54,13 +54,17 @@ def start_drydock():
     ch.setFormatter(formatter)
     logger.addHandler(ch)
 
+
     state = statemgmt.DesignState()
 
     orchestrator = orch.Orchestrator(drydock_provisioner.conf.plugins,
                              state_manager=state)
     input_ingester = ingester.Ingester()
     input_ingester.enable_plugins(drydock_provisioner.conf.plugins.ingester)
 
+    # Now that loggers are configured, log the effective config
+    drydock_provisioner.conf.log_opt_values(logging.getLogger(drydock_provisioner.conf.logging.global_logger_name), logging.DEBUG)
```

Comment:
If there are plans to use oslo_log, this can be done in a far cleaner way. Also, you can use the config option values directly from oslo_config if they're registered properly.